### PR TITLE
Contact Form Section Added

### PR DESCRIPTION
**Pull request by Rana-Mudassir**
Contact Form Section Added with Buttons and textarea. And Styling is done in this branch. With also form spree action.

### CODE REVIEW

**Review by anagudelogu — APPROVED** [review]
Hi Team! @Rana-Mudassir @Nuga0909 @Fahim-Khan-P, 

Looks good to me. 🥇You guys did a great job with the UI design! Congratulations, keep it up. 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  




## Highlights
✅ Great UI design. 🎨 
✅ Clean and organized code. 👍🏼 
✅ GitHub flow used correctly. 💯 
✅ Descriptive PR title and summary. 👍🏼 
✅ Descriptive commit messages. 🥇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -373,6 +373,12 @@ ul {
   border-radius: 8px;
 }
 
+#btn:hover {
+  background: #6070ff;
+  color: #fff;
+  transition: all 0.5s ease-out;
+}
+
```
Comment (Thinus01) [inline style.css:381]: Well done in adding effects to the buttons. Everything seems to be perfect.

**Comment by chigozie1803** [comment]
you have done great job, keep it up.

**Review by Thinus01 — COMMENTED** [review]
Good job with the project. Time to merge it.

**Pull request merged**